markdown/preface.markdown

```diff
@@ -159,102 +160,90 @@ Although this book will focus on NumPy and SciPy,
-beta = u"β"
-```
-
-But in Python 3, *everything* is Unicode:
-
-```python
-β = 0.5
-print(2 * β)
-```
-
-The Python core team decided, rightly, that it was worth supporting characters from all languages as first-class citizens in Python code.
-This is especially true now, when most new coders are from non-English-speaking countries.
-For the sake of interoperability, we still recommend using English characters in most code, but this capability can come in handy, for example, in math-heavy Jupyter notebooks.
-
-Nevertheless, the Python 3 update broke a lot existing code.
-Adding insult to injury, to date, much Python 3 code runs slower than the equivalent Python 2 program — even when many already complained that Python was too slow.
-That's certainly not motivating for someone who has to put work in to make their library work in Python 3, and it's left many continuing to use Python 2.7, refusing to upgrade.
-
-Given the divided state of the community, many developers now write code that is compatible with both Python 2 and 3.
-
-New learners should use Python 3.
-It is the future of the language, and there is no sense in adding to the dead weight of Python 2 code littering the interwebs (including much from your authors!).
-In Elegant SciPy, we use Python 3.5.
-
-If you *must* use Python 2, you can make most of the code in this book
-compatible with Python 2 by using a built-in compatibility module called
-`__future__`, as well as a third-party compatibility library called `six`
-(because 2 times 3 equals six! Get it? Yeah, okay, neither do we).
-You can use them by placing the following imports at the start of the code:
-
-```
-from __future__ import division, print_function
-from six.moves import zip, map, range, filter
-```
-
-A word of warning for Python 2.7 enthusiasts: chapters 5 and 6 rely heavily on the new `@` *matrix multiplication* operator (mentioned above), which is only available in Python 3.5+.
-Trying to use `@` in earlier version of Python will give you a nasty syntax error.
-To make the code in these chapters work with Python 2.7 you will have to use the `.dot` method of NumPy arrays and SciPy matrices, which is decidedly inelegant.
-More on how to do this in chapter 5.
-
-For more reading, check out Ed Schofield's resource, python-future.org,
-and Nick Coghlan's [book-length guide](http://python-notes.curiousefficiency.org/en/latest/python3/questions_and_answers.html) on the transition.
+> **The Great Cataclysm: Python 2 vs. Python 3 {.callout}**
+> 
+> In your Python travels, you may have already heard a few rumblings about which version of Python is better.
+> You may have wondered why it's not just the latest version.
+> 
+> At the end of 2008, the Python core developers released Python 3.0, a
+> major update to Python with better Unicode (international) text
+> handling, type consistency, and streaming data handling, among other
+> improvements.
+> 
+> In most cases, Python 2.6 or 2.7 code cannot be interpreted by Python
```

Comment:
Without Doug Adams, this needs a But, and perhaps the paragraphs can be merged...?